model/host/host.go

```diff
@@ -143,11 +143,8 @@ type Host struct {
 	InstanceTags []Tag `bson:"instance_tags,omitempty" json:"instance_tags,omitempty"`
 
 	AttachVolume bool `bson:"attach_volume" json:"attach_volume"`
-
 	// HomeVolumeGB is the size of the home volume in GB
-	HomeVolumeGB         int    `bson:"home_volume_gb" json:"home_volume_gb"`
-	HomeVolumeID         string `bson:"home_volume_id" json:"home_volume_id"`
-	HomeVolumeDeviceName string `bson:"home_volume_device_name" json:"home_volume_device_name"`
+	HomeVolumeGB int `bson:"home_volume_gb" json:"home_volume_gb"`
```

Comment:
should this be volume size? or stored in the attachment document as well?

Reply:
I renamed to HomeVolumeSize.
The volume size is recorded in the Volumes collection once the volume is created, but I need this field in the HostIntent to persist the size of the volume we're going to ask for in provisioning.